src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -264,9 +263,13 @@ class CosmasGoogleCloudService(private val freeBucketName: String,
             return
         }
         if (versionList.isEmpty()) {
-            val errorStatus = Status.NOT_FOUND.withDescription(
-                    "There is no file in storage with file id ${request.fileId}")
-            LOG.error(errorStatus.description)
+            val description = if (request.startGeneration != -1L) {
+                "There is no file=${request.fileId} in storage"
+            } else {
+                "There is no versions of file=${request.fileId} before generation=${request.startGeneration}"
```

Comment:
will it always print "before generation=-1" ?

Reply:
it should be `request.startGeneration == -1L`, fixed